certs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
I actually believe this line (`!.gitignore`) should be removed, as it will ignore any changes to the .gitignore file itself. I guess you had the "chicken-egg" type of problem, which could've been fixed by `git add -f`:

```sh
jakub@iks:/tmp/qwerty$ echo '*' > .gitignore
jakub@iks:/tmp/qwerty$ git add .gitignore
The following paths are ignored by one of your .gitignore files:
.gitignore
Use -f if you really want to add them.
fatal: no files added
jakub@iks:/tmp/qwerty$ git add -f .gitignore
```

:)